.github/workflows/gradle.yml

```diff
@@ -1,6 +1,9 @@
 name: Airbyte CI
 
-on: push
+on:
+  schedule:
+    - cron: '0 13 * * *'
```

Comment:
1pm UTC

Reply:
Why not every hour?

Reply:
We could do something a little more frequent like 6 hours, but 1 hour will flood the version control channel with notifications in a way that makes it hard to see what changes are going through. Changed it to 6 hours.

Reply:
lol. i think our build is too slow to do once an hour. if you want to wait until green to do something, it only means 50% of every hour will you actually be able to take action.